utils/src/requests/user/update.user.dto.ts

```diff
@@ -0,0 +1,66 @@
+import { IsEmail, IsString, IsOptional, Matches, Min, Max, MaxLength, MinLength } from 'class-validator';
+import { ENG_REGEX, NUM_REGEX, UKR_REGEX, validationMessages, validationOptionsMsg } from '../../validation.util';
+import { ApiPropertyOptional } from '@nestjs/swagger';
+
+export class UpdateUserDTO {
+  @ApiPropertyOptional({
+    description: 'Email of the user',
+  })
+  @IsEmail(undefined, validationMessages.mustBeType('Email', 'an email'))
+  @IsOptional()
+    email?: string;
+
+  @ApiPropertyOptional({
+    description: 'Password of the user',
+  })
+  @IsString(validationMessages.mustBeType('Password', 'a string'))
+  @MinLength(6)
+  @MaxLength(32)
+  @Matches(
+      new RegExp(/^(?=.*[A-Za-z])(?=.*\d).{6,32}$/),
+      validationOptionsMsg('The password must be between 6 and 32 characters long, include at least 1 digit and 1 latin letter'))
```

Comment:
I meant you should also separate the validation message between these 3 decorators for user to clearly know what's the problem. Therefore, you can remove length specification in regex
```suggestion
  @MinLength(6, validationOptionsMsg('Password is too short (min: 6)'))
  @MaxLength(32, validationOptionsMsg('Password is too long (max: 32)'))
  @Matches(
      new RegExp(/^(?=.*[A-Za-z])(?=.*\d)$/),
      validationOptionsMsg('The password must include at least 1 digit and 1 latin letter'))
```